Chore/update trezor common

szymonlesisz opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description

Update trezor-common submodule and build new coins.json

- updated DOGE dust_limit
- removed multiple fee levels for BCH, BTG, DGB
- removed MUE (MonetaryUnit) coin
- removed dead blockbooks (blockchain_link)

related to https://github.com/trezor/trezor-suite/pull/9176

mroz22 commented on packages/connect-common/files/coins.json:
```diff
@@ -2226,7 +2155,23 @@
     ],
     "erc20": [],
-    "eth": [],
+    "eth": [
```
just please manually remove this eth record. this is of course temporary solution, automation script should be improved, I am sure I will get to it one day.

szymonlesisz replied:
ah, ok i did it for you by excluding them completely using -E param

mroz22 reviewed (approved):
just one humble request, otherwise 👍

szymonlesisz commented:
forcepushed and fixed exclusion of "eth" and "erc20" from coins.json. ~~Their absence should not break anything~~ it did so i fixed it in DataManager [1795d4e](https://github.com/trezor/trezor-suite/pull/9188/commits/1795d4ebb901c99129268d2b3cc6113ecd27764a)

mroz22 reviewed (approved):
thanks!
